indico/modules/events/notes/templates/_note.html

```diff
@@ -54,13 +54,21 @@
 {% endmacro %}
 
 
-{% macro _render_note_actions(note, event=none, hidden=true, can_edit=false) %}
+{% macro _render_note_actions(note, event=none, hidden=true, can_edit=false, anchor=none) %}
     <div class="note-actions {% if event %}{{ _togglable_classes(hidden) }}{% endif %}">
         {% if event %}
             <a href="#" class="js-show-note-toggle">{% trans %}Hide{% endtrans %}</a>
         {% endif %}
-        <a href="{{ url_for('event_notes.view', note) }}" target="_blank"
-           class="permalink icon-link" title="{% trans %}Permanent link{% endtrans %}"></a>
+        {% if anchor %}
+            <a href="?note={{ note.id }}#{{ anchor }}" target="_blank"
+            class="permalink icon-link" title="{% trans %}Permanent link{% endtrans %}"></a>
+        {% elif anchor == '' %}
```

Comment:
Also, do we really need to have separate cases for empty and none? When are we using which one right now?

Reply:
The empty string is used to distinguish between the event note and the notes belonging to session blocks. The main event note does not have any anchor element to use, so we just omit it here. Normal notes are anchored to the block titles which have IDs. It doesn't feel very nice but I'm not sure if there is a cleaner alternative..